templates/boot-fastboot/generic-fastboot-boot-template.jinja2

```diff
@@ -6,45 +6,27 @@
 {{ super() }}
 tags:
   - fastboot
-protocols:
-  lava-lxc:
-    name: lxc-{{ platform }}-kernelci-fastboot
-    template: debian
-    distribution: debian
-    release: buster
 {% endblock %}
 {% block actions %}
 actions:
-{%- block deploy_lxc %}
-
-- deploy:
-    namespace: host
-    timeout:
-      minutes: 15
-    to: lxc
-    packages:
-    - android-tools-adb
-    - android-tools-fastboot
-    - cpio
-    - mkbootimg
-{%- endblock %}
-{%- block boot_lxc %}
-
-- boot:
-    namespace: host
-    prompts:
-    - 'root@(.*):/#'
-    timeout:
-      minutes: 5
-    method: lxc
-{%- endblock %}
-{%- block deploy_url %}
 
 - deploy:
     timeout:
       minutes: 40
-    namespace: target
-    to: download
+    to: downloads
+    postprocess:
+      docker:
+        image: montjoie/kci-fastboot
```

Comment:
Please set it to the definitive name, i.e. `kernelci/fastboot`.  We can have a patch on staging to use `kernelci/staging-fastboot` instead.